apps/gov-website/pages/api/identity/judgement.tsx

```diff
@@ -88,9 +88,9 @@ export default withMethodGuard(
 const assignDiscordRole = async (discordUsername: string) => {
```

Comment:
Decided not to move this to `service/discord`?

Reply:
oo forgot about that, cheers